### PR TITLE
Add contributing, supporting and code of conduct md files

### PR DESCRIPTION
**Pull request by anthonymckerron**
Added supporting, code of conduct and contributing markdown files. Re-ordered the Design System page to be in alphabetical order and enforced 150px gap between components.

### CODE REVIEW

File: CODE_OF_CONDUCT.MD [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Code of Conduct
+
+## 1. Purpose
+
+A primary goal of the  design system components is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are
```
Comment (gordongrace) [inline CODE_OF_CONDUCT.MD]: primary goal of the design system sketch file

File: CODE_OF_CONDUCT.MD [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status,
+and religion (or lack thereof).
+
+This code of conduct outlines our expectations for all those who participate in our community, as well as the consequences for unacceptable behavior.
+
+We invite all those who participate and contribute to the design system components to help us create safe and positive experiences for everyone.
```
Comment (gordongrace) [inline CODE_OF_CONDUCT.MD]: ..contribute to this Sketch file...

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+Contributing to the Design-system-sketch-file 🌵
+=================
+
+There are three pages within the sketch file:
```
Comment (gordongrace) [inline CONTRIBUTING.md]:
```suggestion
There are three pages within the Sketch file:
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+##### Design System Page
+
+The purpose of the Design System page is to showcase how different themes affect the look of components. It allows designers to experiment with different colour schemes and see what their designs will look like.
```
Comment (gordongrace) [inline CONTRIBUTING.md]:
```suggestion
The purpose of the Design System page is to demonstrate how different themes affect the look of components. It allows designers to experiment with different colour schemes and see what their designs will look like.
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+##### Symbols Page
+
+The purpose of the Symbols page is to show all of the available components and what they look like in each of their states. It shows what the components will look like in each of the themes (light, dark alt, etc.) while using the standard Design System colours.
```
Comment (gordongrace) [inline CONTRIBUTING.md]:
```suggestion
The purpose of the Symbols page is to show all of the available components and what they look like in each of their states (active, inactive, etc.) in each of the themes (light, dark alt, etc.) while using the standard Design System palette.
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+##### Example Page
+
+The purpose of the Example page is to showcase how an implementation of the Design System would look on different screen types.
```
Comment (gordongrace) [inline CONTRIBUTING.md]:
```suggestion
The purpose of the Example page is to showcase how an implementation of the Design System would look on different screen sizes and orientations.
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+## Adding a new component to the Sketch file.
+----------------------------
+
+When adding a new component, ensure that it is added to the Design System page **and** the Symbols page. When adding a new component, the example page does not have to be updated.
+
+The change log within the Sketch file is no longer updated. Changes are now tracked on Github under the releases section.
```
Comment (gordongrace) [inline CONTRIBUTING.md]: URL?

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+
+The following conventions should be followed when adding a new component to the Sketch file:
```
Comment (gordongrace) [inline CONTRIBUTING.md]:
```suggestion
To aid consistency, the following conventions should be followed when adding a new component to the Sketch file:
```

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+- To maintain a consistent layout for the file, keep components 150 pixels apart from each other.
```
Comment (gordongrace) [inline CONTRIBUTING.md]:
```suggestion
- Maintain spacing of 150 pixels between components.
```

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+- When adding a new component, adhere to the current alphabetical order. Add the new component and shift the other components to the right.
```
Comment (gordongrace) [inline CONTRIBUTING.md]:
```suggestion
- Insert components alphabetically.
```

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+- A component should have at least light and dark versions. Ideally, it should also have a responsive version and an alt version of both.
```
Comment (gordongrace) [inline CONTRIBUTING.md]:
```suggestion
- At a minimum, include light and dark versions. Ideally, it should also have a responsive version and an alt version of both.
```

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+- When naming a component within the Design System Page, the name should follow this style: Component -- Text input. Elements within the component follow this style: Background light alt.
```
Comment (gordongrace) [inline CONTRIBUTING.md]:
```suggestion
- When naming a component within the Design System Page, the name should follow this style: `Component -- Text input`. Elements within the component follow this style: `Background light alt`.
```

Comment (gordongrace) [reply CONTRIBUTING.md]: Should the 'alt' above be capitalised, in order to be consistent with the subsequent comment?

File: CONTRIBUTING.md [hunk 13]
```diff
@@ -0,0 +1,34 @@
...
+- When naming a component within the Symbols page, with the exception of styling declaration of a component (dark, light or alt), all words should be capitalised. For example, Accordion / dark / Alt / Open
+- When adding a component to the Symbols page, the order in which the component versions appear is: _Dark, Dark Alt, Light, Light Alt_.
```
Comment (gordongrace) [inline CONTRIBUTING.md]:
```suggestion
- When adding a component to the Symbols page, use the following order: _Dark, Dark Alt, Light, Light Alt_.
```

File: CONTRIBUTING.md (hunk 12 above)
Comment (anthonymckerron) [reply CONTRIBUTING.md]: Yes it should, applying the change.

File: CONTRIBUTING.md (hunk 7 above)
Comment (anthonymckerron) [reply CONTRIBUTING.md]: Added

**Pull request merged**